test.js

```diff
@@ -34,29 +34,24 @@ const mochaCakes2Rules = {
   ],
 };
 
-const { env, rules, parserOptions } = require(".");
-
 module.exports = {
-  parserOptions,
-  rules: {
-    ...rules,
-    // no only in tests
-    "no-only-tests/no-only-tests": [
-      "error",
-      { block: [ "Feature", "Scenario", "it", "Describe" ] },
-    ],
-    ...mochaCakes2Rules,
-    // chai friendly
-    "no-unused-expressions": 0,
-    "chai-friendly/no-unused-expressions": 2,
-  },
-  plugins: [
-    "no-only-tests",
-    "chai-friendly",
+  plugins: [ "no-only-tests", "chai-friendly" ],
```

Comment:
shouldn't the plugins directive be inside the overrides?

Reply:
It seems both work, but this has been changed in the last commit.